src/Microsoft.CodeAnalysis.Analyzers/VisualBasic/MetaAnalyzers/BasicDiagnosticAnalyzerFieldsAnalyzer.vb

```diff
@@ -8,6 +9,25 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Analyzers.MetaAnalyzers
     <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
     Public Class BasicDiagnosticAnalyzerFieldsAnalyzer
         Inherits DiagnosticAnalyzerFieldsAnalyzer(Of ClassBlockSyntax, StructureBlockSyntax, FieldDeclarationSyntax, TypeSyntax, SimpleAsClauseSyntax)
+
+        Protected Overrides Function IsContainedInFuncOrAction(typeSyntax As TypeSyntax, model As SemanticModel, funcs As ImmutableArray(Of INamedTypeSymbol), actions As ImmutableArray(Of INamedTypeSymbol)) As Boolean
```

Comment:
Can we added VB tests for this scenario?

Reply:
Done. I simplified the check to use [`DelegateInvokeMethod`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.inamedtypesymbol.delegateinvokemethod?view=roslyn-dotnet-4.6.0#microsoft-codeanalysis-inamedtypesymbol-delegateinvokemethod).